flatpak-flutter.yaml

```
@@ -0,0 +1,187 @@
+              type: html
+              url: https://www.freedesktop.org/software/uchardet/releases/
+              version-pattern: uchardet-(\d+\.\d+\.\d+)\.tar\.xz
+              url-template: https://www.freedesktop.org/software/uchardet/releases/uchardet-$version.tar.xz
+      - name: libplacebo
+        buildsystem: meson
+        config-opts:
+          - -Dvulkan=enabled
+          - -Dshaderc=enabled
+        sources:
+          - type: git
+            url: https://code.videolan.org/videolan/libplacebo.git
+            tag: v7.349.0
+            commit: 1fd3c7bde7b943fe8985c893310b5269a09b46c5
+            x-checker-data:
+              type: git
+              tag-pattern: ^v([\d.]+)$
+
+  - name: zenity
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/zenity.git
+        tag: 4.0.3
+        commit: 7f7ac1840cfd914dfac69d947f0849dc50b0f695
+
+  - name: fladder
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/llvm18/bin:/run/build/fladder/flutter/bin
+      env:
+        CC: clang
+        CXX: clang++
+        PUB_CACHE: /run/build/fladder/.pub-cache
+    build-commands:
+      - ls -la /run/build/fladder/
+      
+      # Extract mdk-sdk to FVP's expected directory (use absolute paths)
+      - mkdir -p linux/flutter/ephemeral/.plugin_symlinks/fvp/linux
+      - tar -xf /run/build/fladder/mdk-sdk-linux-x64.tar.xz -C linux/flutter/ephemeral/.plugin_symlinks/fvp/linux
+
+      - sed -i 's|^fvp_setup_deps()|# fvp_setup_deps()|' linux/flutter/ephemeral/.plugin_symlinks/fvp/linux/CMakeLists.txt || true
+       
+      # Verify it's there
+      - ls -la linux/flutter/ephemeral/.plugin_symlinks/fvp/linux/
+      
+      # Build with --no-pub
+      - flutter build linux --release --no-pub
+      
+      - chmod +x install-bundle.sh
+      - ./install-bundle.sh
+      
+      - install -Dm644 flatpak/Fladder.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 icons/fladder_icon.svg /app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - install -Dm644 flatpak/${FLATPAK_ID}.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: script
+        dest-filename: install-bundle.sh
+        commands:
+          - set -e
+          - BUNDLE_DIR=$(find build/linux -type d -name bundle | head -n1)
+          - echo "Found bundle at: $BUNDLE_DIR"
+          - mkdir -p /app/bin
+          - cp -rv "$BUNDLE_DIR"/* /app/bin/
+          - mv /app/bin/Fladder /app/bin/fladder
+          - chmod +x /app/bin/fladder
+      - type: git
+        url: https://github.com/DonutWare/Fladder.git
+        tag: v0.8.0
+        commit: a00b5255992ca4b6d5ba5eb5513e960ed780c198
+      - type: file
+        url: https://sourceforge.net/projects/mdk-sdk/files/nightly/mdk-sdk-linux-x64.tar.xz
+        sha256: fe0116a3b73bfc01dd87d81ba0cd854e33d68ef78699c57deb8d843fea953600
+        dest-filename: mdk-sdk-linux-x64.tar.xz
```

```suggestion
      - type: file
        only-arches:
          - x86_64
        url: https://github.com/microsoft/mimalloc/archive/refs/tags/v2.1.2.tar.gz
        sha256: 2b1bff6f717f9725c70bf8d79e4786da13de8a270059e4ba0bdd262ae7be46eb
        dest: ./build/linux/x64/release
        dest-filename: mimalloc-2.1.2.tar.gz
      - type: file
        only-arches:
          - aarch64
        url: https://github.com/microsoft/mimalloc/archive/refs/tags/v2.1.2.tar.gz
        sha256: 2b1bff6f717f9725c70bf8d79e4786da13de8a270059e4ba0bdd262ae7be46eb
        dest: ./build/linux/arm64/release
        dest-filename: mimalloc-2.1.2.tar.gz
```
As `media_kit_libs_linux` is specified as a git dependency in `pubspec.yaml`, it is not automatically handled by flatpak-flutter, therefore these manual file entries.